features/glance.feature

```diff
@@ -0,0 +1,39 @@
+@glance
+Feature: Glance
+  As an administrator
+  I want to make sure that Glance component is working correctly
+
+  @create_kvm_image
```

Comment:
OK, so I added a review comment to `config/features/glance.yml` - I think this is the wrong choice of tag name.